src/scss/objects/_buttons.scss

```diff
@@ -268,23 +323,37 @@
         }
     }
 
+    .o-btn--fullWidth {
+        display: block;
+        width: 100%;
+
+        // Vertically space out multiple fullWidth buttons
+        // same as .o-btn--fullWidth + .o-btn--fullWidth
+        & + & {
+            margin-top: spacing();
+        }
+    }
+
 
     /**
     * Disabled button
     */
 
-    .o-btn--disabled {
-        cursor: default;
+    .o-btn--disabled,
+    .o-btn.disabled,
+    &.is-disabled,
```

Comment:
Can you use `&` here? I don't think we're nested inside another style unless I'm missing something?

Reply:
From my testing it's applying the style to any element that has the `.is-disabled` class or the `disabled` attribute.

Reply:
Good catch, thanks, fixed!